assists/i.sh

```diff
@@ -0,0 +1,19 @@
+#/usr/bin/dash
+
+# assist: search man pages
+page="$(
+	ls $(
+		man -w |
+		sed 's/:/\/man\* /g'
+	)/man* |
+	sed '/\//d; s/\.gz//g' |
+	awk '!a[$0]++' |\
+	instantmenu -p "       " -q 'search man pages' -l 12 -h -1
+)"
+
+[ -z "$page" ] && exit 0
+
+for i in $page
+do
+	st -e man "$i" &
```

Comment:
```suggestion
	"$HOME/.config/instantos/default/terminal" -e man "$i" &
```

You can't be sure `st` is even installed. The user might have switched it for their favorite terminal emulator.

Note: Wait a while, when a review is in progress, before making changes. It might still be actively changed.